milli/src/update/update_builder.rs

```diff
@@ -1,130 +0,0 @@
-use grenad::CompressionType;
```

Comment:
Have you deleted this file?

Reply:
it was renamed to `indexer_config.rs`, but git hasn't picked it up